phpunit.xml

```diff
@@ -15,5 +15,6 @@
     <env name="CACHE_DRIVER" value="array"/>
     <env name="QUEUE_CONNECTION" value="sync"/>
     <env name="PLATFORM_MW_BACKEND_HOST" value="mediawiki-139-app-backend.default.svc.cluster.default"/>
+    <env name="WBSTACK_SUMMARY_CREATION_RATE_RANGES" value="PT24H,P30D"/>
```

Comment:
It took me quite some time to find information on the time period format that PHP uses here, should we add a comment somewhere in the code mentioning https://www.php.net/manual/en/dateinterval.construct.php or https://en.wikipedia.org/wiki/ISO_8601#Durations ?

Reply:
It's probably best to put such a comment in the chart I guess?